sdcm/utils/threads_utils.py

```diff
@@ -0,0 +1,16 @@
+from threading import Thread
+from repodataParser.RepoParser import Parser
+
+
+class ThreadWithReturnValue(Thread):
```

Comment:
we all ready have 
https://github.com/scylladb/scylla-cluster-tests/blob/00302b48e51729b8ffafb506bd34f6efafd755ef/sdcm/utils/common.py#L277

that can do the same thing, no point of re-implementing it